src/patternfly/components/Select/select.scss

```diff
@@ -0,0 +1,314 @@
+      --pf-c-select__toggle--BorderBottomColor: var(--pf-c-select__toggle--expanded--BorderBottomColor);
+
+      border-bottom-width: var(--pf-c-select__toggle--expanded--BorderBottomWidth);
+    }
+  }
+
+  &.pf-m-typeahead {
+    --pf-c-select__toggle--PaddingTop: 0;
+    --pf-c-select__toggle--PaddingBottom: 0;
+
+    // When the input is the first thing then use a negative left padding so it matches the toggle container
+    input:first-child {
+      margin-left: calc(-1 * var(--pf-c-select__toggle--PaddingLeft));
+    }
+  }
+
+  &.pf-m-plain {
+    &::before {
+      border-color: var(--pf-c-select__toggle--m-plain--BorderColor);
+    }
+
+    &:hover,
+    &.pf-m-hover {
+      &::before {
+        border-color: var(--pf-c-select__toggle--BorderTopColor) var(--pf-c-select__toggle--BorderRightColor) var(--pf-c-select__toggle--hover--BorderBottomColor) var(--pf-c-select__toggle--BorderLeftColor);
+        border-width: var(--pf-c-select__toggle--BorderWidth);
+      }
+    }
+
+    &:active,
+    &.pf-m-active,
+    .pf-m-expanded > & {
+      &::before {
+        border-color: var(--pf-c-select__toggle--BorderTopColor) var(--pf-c-select__toggle--BorderTopColor) var(--pf-c-select__toggle--expanded--BorderBottomColor) var(--pf-c-select__toggle--BorderTopColor);
+        border-width: var(--pf-c-select__toggle--BorderWidth) var(--pf-c-select__toggle--BorderWidth) var(--pf-c-select__toggle--expanded--BorderBottomWidth) var(--pf-c-select__toggle--BorderWidth);
+      }
+    }
+  }
+
+  .pf-c-select-toggle__arrow {
+    margin-left: var(--pf-c-select-toggle__arrow--MarginLeft);
+
+    .pf-m-top.pf-m-expanded > & {
+      transform: rotate(-180deg);
+    }
+  }
+}
+
+.pf-c-select-toggle__text {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+
+.pf-c-select-toggle__wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: flex-start;
+
+  // Max-width is needed for ellipsing the text. This leaves enough space for the toggle caret.
+  // However, for cases where the clear icon button or a badge are added, the correct width will need to be calculated in script
+  max-width: var(--pf-c-select-toggle__wrapper--MaxWidth);
+  white-space: normal;
+
+  // Add space between children
+  > *:not(:last-child),
+  > *:not(:only-child) {
+    margin-right: var(--pf-c-select-toggle__wrapper--not-last-child--MarginRight);
+  }
+
+  // need to reduce the padding, which is driving the height of the chip, so it fits within the toggle
+  .pf-c-chip .pf-c-button {
+    padding-top: var(--pf-c-select-toggle__wrapper--c-chip--c-button--PaddingTop);
+    padding-bottom: var(--pf-c-select-toggle__wrapper--c-chip--c-button--PaddingBottom);
+  }
+}
+
+.pf-c-select-toggle__clear {
+  padding-right: var(--pf-c-select-toggle__clear--PaddingRight);
+}
+
+.pf-c-select-toggle__badge {
+  display: flex;
+  padding-left: var(--pf-c-select-toggle__badge--PaddingLeft);
+}
+
+.pf-c-select-toggle__typeahead {
```

Comment:
`pf-c-select__toggle-typeahead`

Reply:
When you focus in this element, there is a `padding-bottom` and `border-bottom-width` (comes from `.pf-c-form-control`) that change and cause the height to shift a bit. I think the padding is what's causing it to shift.